server/config/db.example.js

```diff
@@ -1,28 +0,0 @@
-module.exports = {
```

Comment:
content's of this file should also not be deleted. 
If using Docker, `.env` configuration take precedence and fallbacks to `db.js` or this file (if `db.js` is not found in `server/config` directory)